yt/yt/ytlib/api/native/client_authentication_impl.cpp

```diff
@@ -15,11 +15,17 @@ using namespace NYTree;
 
 ////////////////////////////////////////////////////////////////////////////////
 
+constexpr int TokenPrefixLength = 10; // "ytct-abcd-"
+
 static TString GenerateToken()
 {
-    constexpr int TokenLength = 16;
-    auto tokenBytes = GenerateCryptoStrongRandomString(TokenLength);
-    auto token = to_lower(HexEncode(tokenBytes.data(), tokenBytes.size()));
+    constexpr int TokenBodyBytesLength = 16;
```

Comment:
Do we need "bytes" here?

Reply:
Yes
We have:

```
constexpr int TokenPrefixLength = 10; // "ytct-abcd-" // 10 characters
constexpr int TokenBodyBytesLength = 16; // 32 characters
constexpr int TokenPrefixBytesLength = 2; // 4 characters
```

Otherwise TokenPrefixLength and TokenBodyLength will have different sense of "length"